hub-config/tasks.json

```diff
@@ -1,25 +1,224 @@
 {
-    "schema_version": "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v3.0.1/tasks-schema.json",
-    "rounds": [
+  "schema_version": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v3.0.1/tasks-schema.json",
+  "rounds": [
+    {
+      "round_id_from_variable": true,
+      "round_id": "nowcast_date",
+      "model_tasks": [
         {
-            "round_id_from_variable": true,
-            "round_id": "origin_date",
-            "model_tasks": [
-                {
-                    "task_ids": {
-                        "origin_date": {
-                            "required": [],
-                            "optional": []
-                        }
-                    },
-                    "output_type": {},
-                    "target_metadata":[]
-                }
-            ],
-            "submissions_due": {
-                "start": "1900-01-01",
-                "end": "2900-12-31"
+          "task_ids": {
+            "nowcast_date": {
+              "required": [
+                "2024-09-11"
+              ],
+              "optional": null
+            },
+            "target_date": {
+              "required": null,
+              "optional": ["2024-08-11", "2024-08-12", "2024-08-13", "2024-08-14", "2024-08-15", "2024-08-16", "2024-08-17", "2024-08-18", "2024-08-19", "2024-08-20", "2024-08-21", "2024-08-22", "2024-08-23", "2024-08-24", "2024-08-25", "2024-08-26", "2024-08-27", "2024-08-28", "2024-08-29", "2024-08-30", "2024-08-31", "2024-09-01", "2024-09-02", "2024-09-03", "2024-09-04", "2024-09-05", "2024-09-06", "2024-09-07", "2024-09-08", "2024-09-09", "2024-09-10", "2024-09-11", "2024-09-12", "2024-09-13", "2024-09-14", "2024-09-15", "2024-09-16", "2024-09-17", "2024-09-18", "2024-09-19", "2024-09-20", "2024-09-21"]
+            },
+            "location": {
+              "required": null,
+              "optional": ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY", "PR"]
+            },
+            "variant": {
```

Comment:
Per other thread, we should change `"variant"` to `"clade"`: https://github.com/reichlab/variant-nowcast-hub/pull/67#issuecomment-2378160501

Reply:
@nickreich Got it--do we also want to make this change in `target_metadata`? i.e., target_id = `clade prop` and `target_name` = `Daily nowcasted clade proportions`?

Reply:
yes

Reply:
@nickreich just pushed this commit!